cob_moveit_config/raw3-1/config/base_controller.yaml

```diff
@@ -0,0 +1,6 @@
+controller_list:
+  - name: base_controller
+    joints:
+      - virtual/x
+      - virtual/y
+      - virtual/theta
```

Comment:
is this used by the MultiDofFollowJointTrajectory plugin? why not use base/x instead of virtual/x?